get_info.pl

```diff
@@ -14,43 +14,22 @@
 sub get_cred
     {
     my $file=shift;
-    my $n=shift;
-    my $type;
-    my $endpt;
-    my $user;
-    my $pass;
-    my $pg_user;
-    my $pg_pass;
-    my $fp;
-    if(open(FP, "<$file"))
-        {
-        my $cnt=0;
-        my $line=<FP>;
-        while($cnt < $n)
-            {
-            $line=<FP>;
-            $cnt=$cnt+1;
-            }
-        if($line =~ /[ \t]*stack[ ]*=[ ]*([^ \t\n]*)[ ]*[,\t]*/) { $type='stack'; $endpt=$1; }
-        if($line =~ /[ \t]*username[ ]*=[ ]*([^ \t\n]*)[ ]*[,\t]*/) { $user=$1; }
-        if($line =~ /[ \t]*password[ ]*=[ ]*([^ \t\n]*)[ ]*[,\t]*/) { $pass=$1; }
-        if($line =~  /[ \t]*pg_user[ ]*=[ ]*([^ \t\n]*)[ ]*[,\t]*/) { $pg_user=$1; }
-        if($line =~  /[ \t]*pg_pass[ ]*=[ ]*([^ \t\n]*)[ ]*[,\t]*/) { $pg_pass=$1; }
-        }
-    else
-        {
-        print "Error: cannot open file \"$file\"\n";
-        exit();
-        }
-    return ($type, $endpt, $user, $pass, $pg_user, $pg_pass);
+    my $creds=undef;
+    my $text=undef;
+    if(open(FP,$file)) { while(my $line=<FP>) { chomp($line); $text=$text.$line; } }
+    print $text;
+    if(defined($text)) { $creds=decode_json($text); }
+    return $creds;
     }
 
 sub get_conn
     {
+    my $db_name=shift;
     my $user=shift;
     my $pass=shift;
 
-    my $conn = DBI->connect("dbi:Pg:dbname=postgres",$user,$pass);
+    print "-->> $db_name, $user, $pass \n";
```

Comment:
Should we be printing the password?

Reply:
@robbaronbu we probably shouldn't be printing the password?

Reply:
good catch.